src/Mod/Fem/CreateLabels.py

```diff
@@ -0,0 +1,147 @@
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "FreeCAD result mechanical task panel"
+__author__ = "PMcB"
+__url__ = "https://www.freecad.org"
+
+## @package view_result_mechanical
+#  \ingroup FEM
+#  \brief create labels
+
+
+import FreeCADGui
+from pivy import coin
+
+"""
+Place Label on the screen with an optionl image.
+positions on screen:
+         (-.9, .9, 0)    top left
+         (.9, .9, 0)     upper right
+         (-.9, -.9, 0)   lower left
+         (.9, -.9, 0)    lower right
+"""
+
+
+class createLabel:
+    def __init__(self, trans, text, image="") -> None:
+        self.textSep = coin.SoSeparator()
+        self.cam = coin.SoOrthographicCamera()
+        self.cam.aspectRatio = 1
+        self.cam.viewportMapping = coin.SoCamera.LEAVE_ALONE
+
+        self.trans = coin.SoTranslation()
+        self.trans.translation = trans
+        self.translation = trans
+        self.color = coin.SoBaseColor()
+        self.color.rgb = (0, 0, 0)
+
+        self.myFont = coin.SoFont()
+        self.myFont.name = "Arial,FreeSans,sans"
+        #       self.myFont.name = "FreeMono,FreeSans,sans"
+        self.size = 20  # 24
+        self.myFont.size.setValue(self.size)
+
+        self.SoText2 = coin.SoText2()
+        self.SoText2.string = text
+
+        self.textSep.addChild(self.cam)
+        self.textSep.addChild(self.trans)
+        self.textSep.addChild(self.color)
+        self.textSep.addChild(self.myFont)
+        self.textSep.addChild(self.SoText2)
+        self.ispicture = False
+        if image != "":
+            self.isimage = True
+            self.myImage = coin.SoImage()
+            self.textSep.addChild(self.myImage)
+            self.myImage.filename.setValue(image)
+        self.activeDoc = FreeCADGui.ActiveDocument
+        self.view = self.activeDoc.ActiveView
+        self.viewer = self.view.getViewer()
+        self.render = self.viewer.getSoRenderManager()
+        self.sup = self.render.addSuperimposition(self.textSep)
+
+    # get position of label
+    def get_position(self):
+        return self.translation
```

Comment:
Probably worth using the python docstring approach:
```python
def get_position(self):
    """Get position of label."""
    return self.translation
```

And doing it for the the rest of the file